src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBindingContext.cs

```diff
@@ -9,14 +10,10 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     /// <summary>
     /// A context that contains operating information for model binding and validation.
     /// </summary>
-    public class ModelBindingContext
+    public class ModelBindingContext : IModelBindingContext
```

Comment:
I thought the reason for the separation was to put the implementation somewhere else... if we're not going to do that that or can't ... might as well have one class.